AUTHORS

```diff
@@ -10,4 +10,4 @@
 
 # Please keep the list sorted.
 
-Vastech SA (PTY) LTD
+GoGraphviz Authors
```

Comment:
I saw in your commit you left `Vastech SA (PTY) LTD` here but changed it in every license, so I figured we should change it here as weel